src/Xamarin.Android.Build.Tasks/Utilities/ResourceDesignerImportGenerator.cs

```diff
@@ -55,7 +58,9 @@ public void CreateImportMethods (IEnumerable<string> libraries)
 						var declaringType = reader.GetTypeDefinition (typeDefinition.GetDeclaringType ());
 						var declaringTypeName = $"{reader.GetString (declaringType.Namespace)}.{reader.GetString (declaringType.Name)}";
 						if (declaringTypeName == resourceDesignerName) {
-							CreateImportFor (declaringTypeName, typeDefinition, method, reader);
+							if (hasAlias)
+								declaringTypeName = $"{alias}.{declaringTypeName}";
```

Comment:
I feel like this *should* be `{alias}::{declaringTypeName}` -- using `::` instead of `.` -- as that's what the [`extern alias` documentation uses](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/extern-alias):

> In the previous example, `GridV1::Grid` would be the grid control from `grid.dll`, and `GridV2::Grid` would be the grid control from `grid20.dll`.

Local testing shows that *both* e.g. `Lib1::Example.A` and `Lib1.Example.A` work, so apparently both are valid.

However, I think that we *do* want `::` and not `.`, so that we "bypass" nested namespace resolution rules.  The reason we use `global::` is so that if someone has a nested `Android` namespace, types still resolve "correctly".